Add PR and issue templates (#479)
open-vsx/publish-extensions: docs/direct_publish_setup.md

# Publishing an extension to OpenVSX on your own

> It is hightly advised that extension authors publish their extensions to OpenVSX as a part of their CI/CD process. See [our reasoning why that is](https://github.com/open-vsx/publish-extensions#when-to-add-an-extension).

Reviewer: ```suggestion
We advise extension authors to publish their extensions to OpenVSX as a part of their CI/CD process. See [our reasoning why that is](https://github.com/open-vsx/publish-extensions#when-to-add-an-extension).
```

> We have a simple template for setting a workflow with GitHub Actions which can help you kickstart your own. This template includes some neat features like:

Reviewer: ```suggestion
To make the process easier, we have provided a simple template of a GitHub Actions workflow to speed up the process of creating your own CI script/process. This template performs the following:
```

- Publishing to GitHub Releases, the Microsoft Marketplace and OpenVSX
- Uploading the `.vsix` package to GitHub Releases as assets

> - Manual triggers that allow you to select what places to release to

Reviewer: ```suggestion
- Manual CI triggers for Open VSX and/or Microsoft Marketplace publishing.
```

Author: @loujaybee I think this new wording does not mention GitHub Releases, which are also a possibility. I am not sure how to rephrase it to still have the `and/or` part in it with three possible options 😄

Reviewer: Updated! Does it work now? 😁

Author: I think this will work, although it does not mention GitHub Releases explicitly, but it is stated two points above, so it isn't all that necessary.

- Automatic triggers from new GitHub Releases

> ## Setup

Reviewer: ```suggestion
## Setup VS Code extension publishing CI workflow
```

> 1. The first step is is to follow the [Publishing Extensions](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions) doc and take note of the access token that you get as you will need it in the next step.

Reviewer: ```suggestion
1. The first step follows the [Publishing Extensions](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions) doc and take note of the access token that you get as you will need it in the next step.
```

> 2. In order to run this the Action above, you need to setup two repository secrets for the Action to use:

Reviewer: ```suggestion
2. To run the GitHub Action above, you need to setup two repository secrets for the Action to use:
```
